tools/annotations.py

```diff
@@ -6,7 +6,7 @@
 
 def extract_annotated_types(file_path, annotation, clang_args):
     index = clang.cindex.Index.create()
-    translation_unit = index.parse(file_path, args=clang_args)
+    translation_unit = index.parse(file_path, args=clang_args, options=clang.cindex.TranslationUnit.PARSE_INCOMPLETE | clang.cindex.TranslationUnit.PARSE_SKIP_FUNCTION_BODIES)
```

Comment:
this is another leftover. It's part of an investigation I was doing with bindings (but turns out bindgen is more sensitive) and macros that don't always seem to play well in all cases.